libsrc/apple2/beep.s

```diff
@@ -0,0 +1,25 @@
+;
+; Colin Leroy-Mira, 2024
+;
+; void beep(void)
+;
+
+        .export         _beep
+
+        .include        "apple2.inc"
+
+        .segment        "LOWCODE"
+
+_beep:
+        ; Switch in ROM and call BELL
+        bit     $C082
+
+        lda     CH              ; Bell scrambles CH in 80col mode on IIgs, storing
+        pha                     ; it in OURCH and resetting CH to 0. Save it.
+        jsr     $FF3A           ; BELL
+        pla
+        sta     CH              ; Restore CH
+
+        ; Switch in LC bank 2 for R/O and return
+        bit     $C080
+        rts
```

Comment:
Sorry to nitpick once more, but I'd like to have this done like in:
https://github.com/cc65/cc65/blob/master/libsrc/apple2/clrscr.s
https://github.com/cc65/cc65/blob/master/libsrc/apple2/home.s

In other words: There's always the ROM banking wrapper named like the ROM symbol (here `BELL`) and then building on this there's any library code (even it's only an alias like clrscr()).